controller/src/main/java/io/pravega/controller/server/ControllerServiceStarter.java

```diff
@@ -186,6 +184,10 @@ protected void startUp() {
             log.info("Creating the checkpoint store");
             checkpointStore = CheckpointStoreFactory.create(storeClient);
 
+            // Initialize Stream and Transaction metrics.
```

Comment:
why do we need to do this initialization? 
Upon its first usage, it would get initialized once and reused everywhere.

Reply:
Although i think its better for the readability of the code.

Reply:
Yes, to make the initialization explicit I invoked it there.